src/models/logic/transaction.ts

```diff
@@ -430,6 +435,50 @@ async function handleUTXO(txInst: TransactionInstance, blockNumber: number) {
                 })
             );
         }
+        const orders = await transferAsset.getOrders();
+        if (orders) {
+            await Promise.all(
+                orders.map(
+                    async (
+                        orderOnTransferInst: OrderOnTransferInstance,
+                        transactionOrderIndex: number
+                    ) => {
+                        const orderInst = await orderOnTransferInst.getOrder();
+                        if (!orderInst) {
+                            throw Exception.InvalidUTXO;
+                        }
+                        const order = orderInst.get({ plain: true });
+                        const orderHash = new H256(order.orderHash);
+                        const assetType = new H160(order.assetTypeFrom);
+                        const shardId = order.shardIdFrom;
+                        const lockScriptHash = new H160(
+                            order.lockScriptHashFrom
+                        );
+                        const parameters = order.parametersFrom;
+                        const quantity = new U64(order.assetQuantityFrom);
+                        const owner = getOwner(
+                            lockScriptHash,
+                            parameters,
+                            networkId
+                        );
+                        return createUTXO(
```

Comment:
Could you explain to me about why we should create UTXOs from an order?
I think that the order only specifies which inputs/outputs are exchanged by this order. I don't know how a new UTXO could be created by order.

Reply:
I was wrong about UTXO. It seems no handling is need for orders, I'll remove above part.